### ISSUE TITLE
Can i Install Truevote Backend program on a Ubuntu Machine 

### ISSUE THREAD

**Issue by akolaarthurali** [TrueVote/TrueVote.Api#80]
From the readme file have just read Truevote is written in C#, a programming language widely for developing Windows-based systems. 

I extensively use Linux in particular ubuntu for my deployments. I loved everything about Truevote, its proposition, the founders behind this project (the wide programming scope) and more so because Truevote uses bitcoin to secure electoral ballots using opentimestamps(I just learnt 😳 of it from reading about Truevote).

I want to run this project on ubuntu how I can get to do that.

**Comment by morrisonbrett** [comment]
Yes, the back-end services are written in C# .NET Core 9.0. .NET Core can run in Windows, Linux, macOS, and Docker. The current deployed instances of TrueVote services are actually running in Ubuntu in production on Microsoft Azure.

**Comment by akolaarthurali** [comment]
Thanks very much Brett for the feedback back.. I look forward to deploying an instance.  I will rather prefer to use digitalocean.com 😁 to Microsoft Azure.😁

And actually I also got myself to watch a "coin stores" podcast where you were hosted by Natalie... indeed you fully disclosed in there how the current instance is deployed on Microsoft Azure. I also loved how you pitched truevote.org to us "solving the inconvenience that comes with elections and secondly the full transparency of the elections results". I really love ❤️ the problem/challenges truevote solves and more importantly how the code is open source and that we launch our own instances
